_data/gsoc-ideas.yml

```
@@ -25,7 +25,7 @@
   prereqs: Scala, ideally experience with serverless
   difficulty: Medium.
   length: medium
-  mentors: [armanbilge, bpholt]
+  mentors: [armanbilge, bpholt, chingles]
```

This one is your GitHub handle :)

```suggestion
  mentors: [armanbilge, bpholt, Chingles2404]
```

---

Thank you! I erroneously assumed it was based on the Typelevel authors HAHA